src/GUI/mainwindow.ui

```diff
@@ -1077,8 +1077,15 @@
     <addaction name="action_print_direction"/>
     <addaction name="action_print_spacing"/>
    </widget>
+   <widget class="QMenu" name="menuView">
+    <property name="title">
+     <string>View</string>
+    </property>
+    <addaction name="action_show_credits"/>
+   </widget>
    <addaction name="menu_file"/>
    <addaction name="menu_help"/>
+   <addaction name="menuView"/>
```

Comment:
Don't think we need a menu tab for this.

This fits (somewhat) under Help (which already links to GitHub, which has our names).

Reply:
That's fair but I plan to add more under view.

Reply:
Cool!

If making any styling changes, just added some information to the Trello card about that.